src/porting_android.cpp

```diff
@@ -359,19 +359,16 @@ bool upgrade(const std::string &item)
 	return res == JNI_TRUE;
 }
 
-int getRoundScreen()
+int getWindowSafeArea()
 {
-	static const int radius = [](){
-		jmethodID getRadius = jnienv->GetMethodID(activityClass,
-				"getRoundScreen", "()I");
+	SDL_Window* window = SDL_GetKeyboardFocus();
```

Comment:
Maybe
```
const video::SExposedVideoData exposedData = driver->getExposedVideoData();
SDL_Window *window = exposedData.OpenGLSDL.Window;
```
?

Or move it to rendering engine if no access to irrlicht driver from here. SDL_GetKeyboardFocus() seems bad idea.